app/utils/database.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import urlParse from 'url-parse';
+
+import DatabaseConnectionException from '@database/exceptions/database_connection_exception';
+import DatabaseManager from '@database/manager';
+
+type SetActiveDatabaseArgs = {
+  serverUrl: string;
+  displayName?: string;
+};
+
+export const createAndSetActiveDatabase = async ({serverUrl, displayName}: SetActiveDatabaseArgs) => {
```

Comment:
do we need any of this? I believe this is already achievable with the DBManager Instance